test/ci/shellcheck.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# TODO: add possibility to run this script directly
```

Comment:
I think ideally it would be possible to run shellcheck with our `run-tests.sh` script and include it in `all()`.